pkg/trust/policy.go

```diff
@@ -54,14 +56,23 @@ type genericRepoMap map[string]json.RawMessage
 
 // DefaultPolicyPath returns a path to the default policy of the system.
 func DefaultPolicyPath(sys *types.SystemContext) string {
+	if sys != nil && sys.SignaturePolicyPath != "" {
+		return sys.SignaturePolicyPath
+	}
+
+	confDir, _ := homedir.GetConfigHome()
```

Comment:
GetConfigHome [interprets](https://github.com/containers/storage/blob/c929d2050e4c1fb3c7a56b28230675d908bc8aeb/pkg/homedir/homedir.go#L14) `$XDG_CONFIG_HOME`, so that would again be a divergence in behavior.

I don’t have a strong opinion on which of the two behaviors should be used — it’s again the same thing, that diverging seems not the right thing to do in this PR, and that changes in behavior should happen across all three locations.

Reply:
Ok, I just reverted this specific change. IMHO, I think the same as you. I might not be the correct way of doing it, but at least we keep the same behaviour, and changing the behaviour everywhere is beyond the scope of this PR.